src/libs/ComposerUtils/updateNumberOfLines/types.ts

```diff
@@ -1,6 +1,7 @@
 import {NativeSyntheticEvent, TextInputContentSizeChangeEventData} from 'react-native';
 import ComposerProps from '@libs/ComposerUtils/types';
+import Styles from '@styles/styles';
 
-type UpdateNumberOfLines = (props: ComposerProps, event: NativeSyntheticEvent<TextInputContentSizeChangeEventData>) => void;
+type UpdateNumberOfLines = (props: ComposerProps, event: NativeSyntheticEvent<TextInputContentSizeChangeEventData>, styles: typeof Styles) => void;
```

Comment:
not sure if this type is ok, @fabioh8010 wdyt?

Reply:
what's wrong here?

Reply:
shouldn't be `typeof styles` if so?

Reply:
ah, it's just a naming convention thing. Changing it now, per Micheal's request